File: Dia55/ValidadorMails.js
// Validador de Emails en JavaScript

const REGEX = /^\w+@\w+\.\w+$/

function isValidEmail(email) {
  return REGEX.test(email)
}

console.log(isValidEmail('name@example.com')) // true
console.log(isValidEmail('name.example@com')) // false

File: Dia55/ValidadorMails.py
# Validador de Emails en Python

import re

REGEX = r'^\w+@\w+\.\w+$'

def isValidEmail(email):
  return re.match(REGEX, email) != None


print(isValidEmail('name@example.com')) # True
print(isValidEmail('name.example@com')) # False